README.md

```diff
@@ -1,9 +1,10 @@
 # amaps
-implementation of nlmaps for Amsterdam (for testing)
+implementation of nlmaps for Amsterdam
 
-This repo contains a reference implementation of nlmaps with Amsterdam's extensions and styling, for testing purposes. The goal is to demonstrate correct working of nlmaps in an application using browser tests. For code unit testing, see the [nlmaps](https://github.com/webmapper/nlmaps) repository.
+This repository builds `nlmaps` with a configuration file for Amsterdam, specifying Amsterdam's map styling and map layers. In addition, this repository contains several wrapper scripts which bundle the resulting `nlmaps` build with functionality for specific use cases, like querying certain API's when the map is clicked. These specific cases are:
 
-The repo also contains a small library of functions to ease Amsterdam's specific usage of `nlmaps`. This is found in `src`.
+* MORA (meldingen openbare ruimte)
+* TVM (tijdelijke parkeersmaatregelen).
```

Comment:
Is het niet 'tijdelijke *ver*keersmaatregelen'? @rmaijers

Reply:
yes (leuke verhaspeling)

Reply:
ehe. Fix gepushed.